drivers/hmis/app/models/hmis/project_unit_type_mapping.rb

```diff
@@ -17,32 +17,72 @@ class Hmis::ProjectUnitTypeMapping < Hmis::HmisBase
   scope :active, -> { where(active: true) }
 
   def self.freshen_project_units(user:, today: Date.current)
+    scope = preload(:project, :unit_type).order(:id).to_a
+    create_new_units(scope, user: user)
+    destroy_inactive_units(scope, today: today)
```

Comment:
```suggestion
    scope = preload(:project, :unit_type).order(:id)
    # Create initial Units for new Active Project Unit Type Mappings
    create_new_units(scope.active, user: user)
    # Destroy Units for Project Unit Type Mappings that are Inactive
    destroy_inactive_units(scope.inactive, today: today)
```

Some minor suggestions for readability-
- add comments
- pass only relevant scope (active vs inactive)
- pass scope instead of array- since the other functions accept a variable name `scope` I would expect it to be an activerecord scope not an array. Don't think it matters much either way for perf since it's a relatively small set